src/main/java/eu/enmeshed/model/relationships/RelationshipStaus.java

```diff
@@ -0,0 +1,18 @@
+package eu.enmeshed.model.relationships;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum RelationshipStaus {
```

Comment:
Ahh, here is the "RelationshipStaus" defined :D Better than comment on every usage of it.

Reply:
Fixed.